composer.json

```diff
@@ -88,6 +88,39 @@
     "bin": [
         "redaxo/bin/console"
     ],
+    "scripts": {
+        "clone-addon": ".tools/bin/clone-addon",
+        "refresh": ".tools/bin/refresh",
+        "release": ".tools/bin/release",
+        "update-all": ".tools/bin/update-requirements",
+
+        "cs": "php-cs-fixer fix -v --ansi",
+        "phpunit": "phpunit --colors=always",
+        "phpstan": "phpstan analyse --ansi",
```

Comment:
Normalerweise sehe ich automatisch farbige Ausgaben. Über composer griff der Automatismus aber nicht.
@staabm Weißt du, ob die Variante so auf Windows funktioniert? Hängt vermutlich von der Shell ab, oder?

Reply:
Ich persönlich hab auf windows kein problem mit farbigen codes

Reply:
Ggf macht es auch sinn nen composer issue aufzumachen

Reply:
https://github.com/composer/composer/issues/5606

Reply:
ohh frisch aus der Presse, heute nen blogpost gefunden:
https://tomasvotruba.com/blog/2020/09/28/how-to-add-colors-to-continuious-integration-output/